examples/dt/compute/amd-sev/edpm/openstackdataplanenodeset.yaml

```diff
@@ -0,0 +1,134 @@
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-amd-sev
+spec:
+  env:
+    - name: ANSIBLE_FORCE_COLOR
+      value: "True"
+  services:
+    - bootstrap
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - libvirt
+    - ovn
+    - neutron-sriov
```

Comment:
This can be removed since sr-iov is not required for amd-sev

Reply:
ok